packages/api/amplify_api_dart/test/web_socket/web_socket_bloc_test.dart

```diff
@@ -307,6 +310,149 @@ void main() {
       mockPollClient.induceTimeout = false;
     });
 
+    test('should reconnect when process unpauses', () async {
```

Comment:
small-nit: pause is referenced throughout the tests. I think it works in some places more than others. Feel free to update as you see fit. 

```suggestion
    test('should reconnect when process resumes', () async {
```

Reply:
Updated